cli/lib/context/context_open_in_browser.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:io';
+
+import 'package:stax/context/context.dart';
+
+extension ContextOpenInBrowser on Context {
+  void openInBrowser(String url) {
```

Comment:
Let's have it return external command, so the user will have control over `announce`, `runSync`, and other options of external commands.
```suggestion
  ExternalCommand openInBrowser(String url) {
```